database/dbEvents.php

```diff
@@ -161,6 +162,15 @@ function fetch_events_on_date($date) {
     return $events;
 }
 
+function make_field($result_row){
```

Comment:
Consider adding a comment explaining the function and its purpose

Reply:
Will do!